Dockerfile

```diff
@@ -17,8 +17,11 @@ RUN cd brainz-mmd2-jaxb && \
 
 COPY ./mmd-schema/brainz-mmd2-jaxb brainz-mmd2-jaxb
 COPY ./mb-solr mb-solr
-RUN cd brainz-mmd2-jaxb && \
+COPY ./mbsolr-jaxb mbsolr-jaxb
+RUN --mount=type=cache,target=/root/.m2 cd brainz-mmd2-jaxb && \
     mvn install && \
+    cd ../mbsolr-jaxb && \
+    mvn assembly:single && \
```

Comment:
I tried to configure the `maven-assembly-plugin` in the pom.xml for mbsolr-jaxb to automatically create the "jar-with-dependencies" file when running "mvn package", but it didn't work. I'm not sure if this is because we have no source code, and it doesn't import from one of the dependencies?

In any case, running the "assembly:single" target manually will successfully make the jar, so I decided to leave it like this. This does mean that the plugin configuration in the pom is a bit deceptive, as someone reading it might assume that they can run `mvn package` to do the same thing. Perhaps we can remove it to reduce confusion